doc/md/versioned-migrations.md

```diff
@@ -174,3 +174,40 @@ versioned migration, you need to take some extra steps.
    In case of `golang-migrate` this can be done by forcing your database version as
    described [here](https://github.com/golang-migrate/migrate/blob/master/GETTING_STARTED.md#forcing-your-database-version).
 
+## Use a custom formatter
+
+It is possible to use another tool for migration management instead of `golang-migrate/migrate`. For example, you can define and use the following formatter to use [pressly/goose](https://github.com/pressly/goose).
+
+```go
+var (
+	templateFuncs = template.FuncMap{
+		"now": time.Now,
+		"sem": ensureSemicolonSuffix,
+		"rev": reverse,
+	}
+	// gooseFormatter is an implementation for compatible formatter with goose.
+	// https://github.com/pressly/goose/blob/205faf5cd314dcc76f2687bac7cfe0534a4bd5a9/create.go#L90-L99
```

Comment:
I am not sure, why you link that part of the repo. Do you think it is needed?

Reply:
I wrote this just in case the format definition for original is in the link. Perhaps it is not necessary.